src/engine/controls/loopingcontrol.cpp

```diff
@@ -166,6 +166,18 @@ LoopingControl::LoopingControl(const QString& group,
             this, &LoopingControl::slotBeatJump, Qt::DirectConnection);
     m_pCOBeatJumpSize = new ControlObject(ConfigKey(group, "beatjump_size"),
                                           true, false, false, 4.0);
+
+    m_pCOBeatJumpSizeHalve = new ControlPushButton(ConfigKey(group, "beatjump_size_halve"));
```

Comment:
The delete call is missing if you wish to match the existing code. 
A better solution would be using a unique_ptr to manage the live-time

Reply:
for now, I added the delete call and a TODO for unique_ptr.

I could change all controls here to unique_ptr, though, how would I deal with the controls in lists like `QList<BeatLoopingControl*> m_beatLoops;`? They'd remain regular pointers, right?

Reply:
QList does not support unique_ptr we may change to std::vector to work around this.